### PR TITLE
Id: Added Indonesian Translations

### PR DESCRIPTION
**Pull request by willysr**
- [X] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/gh-pages/CONTRIBUTING.md)?
- [X] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
Thanks so much for your contribution! Without people like you submitting PRs we couldn't run this project. You rock, @willysr!

**Comment by willysr** [comment]
Thanks for the initial effort!!!

File: _data/locales/id.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+id:
+  locale_name: Indonesian
```
Comment (MikeMcQuaid) [inline _data/locales/id.yml]: @willysr Actually: just noticed this. Could you update this so it's the Indonesian name for Indonesian? Thanks!

Comment (willysr) [reply _data/locales/id.yml]: sure, i will send another PR in a minute

thanks

Comment (willysr) [reply _data/locales/id.yml]: sent as https://github.com/github/opensource.guide/pull/547

**Pull request merged**